RELEASE-PLAN.md

```diff
@@ -0,0 +1,53 @@
+# Background and Context
+
+Releases of the Pathfinder Framework (Version 2 can be found [here](https://www.carbon-transparency.com/media/jpslsujn/pathfinder-framework.pdf)) are strongly
+informing the tech specs releases.
+
+Therefore, this timeline page contains the release plan for both, the Pathfinder Framework and the Pathfinder Tech Specs.
+
+In addition, the tables below disclose deprecation of releases and subsequent sunset dates.
+
+## Definitions
+
+<dl>
+<dt>Draft Release</dt>
+<dd>The release of draft document, starting a period of feedback collection from the community</dd>
+<dt>Deprecation</dt>
+<dd>Companies are no longer encouraged to use the release (e.g. to compute and subsequently publish PCFs based on it)</dd>
+<dt>Sunset</dt>
+<dd>The release is no longer supported. Companies are discouraged to use the release (e.g. to compute and to subsequently publish PCFs based on it)</dd>
+</dl>
+
+
+
+**As always, the timelines are subject to change.**
+
+## Pathfinder Framework: Upcoming Releases
+
+| When | Milestone | What |
+| --- | --- | --- |
+| December 31, 2023 | Deprecation | Pathfinder Framework v1 |
+| Mid-May to Mid-June 2024 | Draft Release | Pathfinder Framework v2.1 |
+| Q3 2024 | Published Release | Pathfinder Framework v2.1 |
+| Q3 2024 (Same date as release of updated Pathfinder Framework (v2.1)) | Deprecation | Pathfinder Framework v2.1 |
+| October 1, 2024 | Sunset | Pathfinder Framework v1 |
+
+### Deprecation and Sunsetting of Pathfinder Framework 
+
+| Framework | When | Deprecation | Sunsetting |
+| --- | --- | --- | --- |
+| Version 1 | June 2022 | Dec 2023 | Oct 2024 |
+| Version 2 | Jan 2023 | Q3 2024 (when Version 2 update is released) | June 2025 |
+| Version 2 update | Q3 2024 | TBD | TBD |
+
+## Tech Specs Releases
+
+| When | Milestone | What |
+| --- | --- | --- |
+| November 9, 2023 | Draft Release | Tech Specs v2.1.0  |
```

Comment:
For completeness I would also add in here the previous releases. 

I have also heard feedback from our community that it is difficult to find the URL to each release from the tech specs. The links at the top (re "This version" and "Latest published version" help but are not sufficient to locate all releases. Right now for example I can't find the link to v1.0.1  Therefore would be helpful to include the url within this Release Plan of historic releases...such as something like the below?
| ??? | Published Release | Tech Specs v1.0.1  |
| February 21, 2023 | Published Release | [Tech Specs v2.0.0](https://wbcsd.github.io/tr/2023/data-exchange-protocol-20230221/)  |

Reply:
> I have also heard feedback from our community that it is difficult to find the URL to each release from the tech specs.

Agreed, this is a long-standing issue indeed. Let's talk about this later today

Reply:
Added links to the past releases